extensions/figma-files/src/menu-bar.tsx

```diff
@@ -39,12 +49,28 @@ export default function Command() {
         },
       }}
       tooltip="Figma files"
-      isLoading={isLoadingVisitedFiles || isLoading}
+      isLoading={isLoadingVisitedFiles || isLoading || isLoadingStarredFiles}
     >
-      {error && <MenuBarExtra.Item title="Error" key="ErrorState" />}
+      {(error || starredFilesError) && <MenuBarExtra.Item title="Error" key="ErrorState" />}
+      {starredFiles && (
+        <>
```

Comment:
Don't think you need the fragment here since there is only one child.

Reply:
Didn't get this part @michaelwschultz, can you explain?

Reply:
Ah makes sense. fixed it!